### ISSUE TITLE
19.1.0 is not able to migrate from older DB version, stuck forever starting up

### ISSUE THREAD

**Issue by benapetr** [ansible/awx#10189]
##### ISSUE TYPE
 - Bug Report

##### ENVIRONMENT
* AWX version: 19.1.0
* AWX install method: docker
* Ansible version:  X.Y.Z
* Operating System: CentOS 7
* Web Browser: irrelevant

##### STEPS TO REPRODUCE

Run make docker-compose against older version, in my case 17.0.0

##### EXPECTED RESULTS

It works

##### ACTUAL RESULTS

Migration starts, connection to DB is succesful, then in one step it throws exception, and continues, then nginx daemon restarts forever saying that migration is still in progress, see logs

##### ADDITIONAL INFORMATION

```
awx_1_1     | Operations to perform:
awx_1_1     |   Apply all migrations: auth, conf, contenttypes, main, oauth2_provider, sessions, sites, social_django, sso, taggit
awx_1_1     | Running migrations:
awx_1_1     |   Applying conf.0009_rename_proot_settings... OK
awx_1_1     |   Applying main.0124_execution_environments... OK
awx_1_1     |   Applying main.0125_more_ee_modeling_changes... OK
awx_1_1     |   Applying main.0126_executionenvironment_container_options... OK
awx_1_1     |   Applying main.0127_reset_pod_spec_override... OK
awx_1_1     |   Applying main.0128_organiaztion_read_roles_ee_admin... OK
awx_1_1     |   Applying main.0129_unifiedjob_installed_collections... OK
awx_1_1     |   Applying main.0130_ee_polymorphic_set_null... OK
awx_1_1     |   Applying main.0131_undo_org_polymorphic_ee... OK
awx_1_1     |   Applying main.0132_instancegroup_is_container_group... OK
awx_1_1     |   Applying main.0133_centrify_vault_credtype... OK
awx_1_1     |   Applying main.0134_unifiedjob_ansible_version... OK
awx_1_1     |   Applying main.0135_schedule_sort_fallback_to_id... OK
awx_1_1     |   Applying main.0136_scm_track_submodules... OK
awx_1_1     | 2021-05-13 13:16:58,692 INFO     [-] awx.main.migrations deleted (137747, {'main.ActivityStream_unified_job': 0, 'main.UnifiedJob_dependent_jobs': 0, 'main.UnifiedJob_notifications': 0, 'main.UnifiedJob_labels': 0, 'main.UnifiedJob_credentials': 0, 'main.JobLaunchConfig_credentials': 0, 'main.ActivityStream_inventory_update': 0, 'main.InventoryUpdateEvent': 130823, 'main.InventoryUpdate': 2308, 'main.JobLaunchConfig': 2308, 'main.UnifiedJob': 2308})
awx_1_1     | 2021-05-13 13:16:58,693 INFO     [-] awx.main.migrations Deleted 2308 custom inventory script sources.
awx_1_1     |   Applying main.0137_custom_inventory_scripts_removal_data...Traceback (most recent call last):
awx_1_1     |   File "/usr/local/bin/awx-manage", line 9, in <module>
awx_1_1     |     load_entry_point('awx', 'console_scripts', 'awx-manage')()
awx_1_1     |   File "/awx_devel/awx/__init__.py", line 164, in manage
awx_1_1     |     execute_from_command_line(sys.argv)
awx_1_1     |   File "/var/lib/awx/venv/awx/lib64/python3.8/site-packages/django/core/management/__init__.py", line 381, in execute_from_command_line
awx_1_1     |     utility.execute()
awx_1_1     |   File "/var/lib/awx/venv/awx/lib64/python3.8/site-packages/django/core/management/__init__.py", line 375, in execute
awx_1_1     |     self.fetch_command(subcommand).run_from_argv(self.argv)
awx_1_1     |   File "/var/lib/awx/venv/awx/lib64/python3.8/site-packages/django/core/management/base.py", line 323, in run_from_argv
awx_1_1     |     self.execute(*args, **cmd_options)
awx_1_1     |   File "/var/lib/awx/venv/awx/lib64/python3.8/site-packages/django/core/management/base.py", line 364, in execute
awx_1_1     |     output = self.handle(*args, **options)
awx_1_1     |   File "/var/lib/awx/venv/awx/lib64/python3.8/site-packages/django/core/management/base.py", line 83, in wrapped
awx_1_1     |     res = handle_func(*args, **kwargs)
awx_1_1     |   File "/var/lib/awx/venv/awx/lib64/python3.8/site-packages/django/core/management/commands/migrate.py", line 232, in handle
awx_1_1     |     post_migrate_state = executor.migrate(
awx_1_1     |   File "/var/lib/awx/venv/awx/lib64/python3.8/site-packages/django/db/migrations/executor.py", line 117, in migrate
awx_1_1     |     state = self._migrate_all_forwards(state, plan, full_plan, fake=fake, fake_initial=fake_initial)
awx_1_1     |   File "/var/lib/awx/venv/awx/lib64/python3.8/site-packages/django/db/migrations/executor.py", line 147, in _migrate_all_forwards
awx_1_1     |     state = self.apply_migration(state, migration, fake=fake, fake_initial=fake_initial)
awx_1_1     |   File "/var/lib/awx/venv/awx/lib64/python3.8/site-packages/django/db/migrations/executor.py", line 245, in apply_migration
awx_1_1     |     state = migration.apply(state, schema_editor)
awx_1_1     |   File "/var/lib/awx/venv/awx/lib64/python3.8/site-packages/django/db/migrations/migration.py", line 124, in apply
awx_1_1     |     operation.database_forwards(self.app_label, schema_editor, old_state, project_state)
awx_1_1     |   File "/var/lib/awx/venv/awx/lib64/python3.8/site-packages/django/db/migrations/operations/special.py", line 190, in database_forwards
awx_1_1     |     self.code(from_state.apps, schema_editor)
awx_1_1     |   File "/awx_devel/awx/main/migrations/_inventory_source.py", line 105, in delete_custom_inv_source
awx_1_1     |     ct, deletions = InventorySource.objects.filter(source='custom').delete()
awx_1_1     |   File "/var/lib/awx/venv/awx/lib64/python3.8/site-packages/django/db/models/query.py", line 710, in delete
awx_1_1     |     collector.collect(del_query)
awx_1_1     |   File "/var/lib/awx/venv/awx/lib64/python3.8/site-packages/django/db/models/deletion.py", line 224, in collect
awx_1_1     |     field.remote_field.on_delete(self, field, sub_objs, self.using)
awx_1_1     |   File "/var/lib/awx/venv/awx/lib64/python3.8/site-packages/django/db/models/deletion.py", line 15, in CASCADE
awx_1_1     |     collector.collect(sub_objs, source=field.remote_field.model,
awx_1_1     |   File "/var/lib/awx/venv/awx/lib64/python3.8/site-packages/django/db/models/deletion.py", line 224, in collect
awx_1_1     |     field.remote_field.on_delete(self, field, sub_objs, self.using)
awx_1_1     |   File "/awx_devel/awx/main/utils/polymorphic.py", line 18, in SET_NULL
awx_1_1     |     return models.SET_NULL(collector, field, sub_objs.non_polymorphic(), using)
awx_1_1     | AttributeError: 'QuerySet' object has no attribute 'non_polymorphic'
awx_1_1     | make: *** [Makefile:188: migrate] Error 1
awx_1_1     | if [ "/var/lib/awx/venv/" ]; then \
awx_1_1     | 	. /var/lib/awx/venv//awx/bin/activate; \
awx_1_1     | fi; \
awx_1_1     | awx-manage provision_instance --hostname=awx_1; \
awx_1_1     | awx-manage register_queue --queuename=tower --instance_percent=100;
awx_1_1     | Successfully registered instance awx_1
awx_1_1     | (changed: True)
awx_1_1     | Instance Group already registered tower
awx_1_1     | Default Execution Environment(s) registered.
awx_1_1     | (changed: True)
awx_1_1     | /usr/local/lib/python3.6/site-packages/supervisor/options.py:475: UserWarning: Supervisord is running as root and it is searching for its configuration file in default locations (including its current working directory); you probably want to specify a "-c" argument specifying an absolute path to a configuration file for improved security.
awx_1_1     |   'Supervisord is running as root and it is searching '
awx_1_1     | 2021-05-13 13:17:10,039 CRIT Supervisor is running as root.  Privileges were not dropped because no user is specified in the config file.  If you intend to run as root, you can set user=root in the config file to avoid this message.
awx_1_1     | 2021-05-13 13:17:10,044 INFO RPC interface 'supervisor' initialized
awx_1_1     | 2021-05-13 13:17:10,044 CRIT Server 'unix_http_server' running without any HTTP authentication checking
awx_1_1     | 2021-05-13 13:17:10,045 INFO supervisord started with pid 105
awx_1_1     | 2021-05-13 13:17:11,048 INFO spawned: 'stdout' with pid 108
awx_1_1     | 2021-05-13 13:17:11,050 INFO spawned: 'awx-dispatcher' with pid 109
awx_1_1     | 2021-05-13 13:17:11,052 INFO spawned: 'awx-receiver' with pid 110
awx_1_1     | 2021-05-13 13:17:11,055 INFO spawned: 'awx-uwsgi' with pid 111
awx_1_1     | 2021-05-13 13:17:11,058 INFO spawned: 'awx-daphne' with pid 112
awx_1_1     | 2021-05-13 13:17:11,061 INFO spawned: 'awx-nginx' with pid 113
awx_1_1     | 2021-05-13 13:17:11,064 INFO spawned: 'awx-wsbroadcast' with pid 114
awx_1_1     | 2021-05-13 13:17:11,068 INFO spawned: 'awx-rsyslogd' with pid 116
awx_1_1     | 2021-05-13 13:17:11,071 INFO spawned: 'awx-receptor' with pid 117
awx_1_1     | 2021-05-13 13:17:11,147 INFO exited: awx-nginx (exit status 2; not expected)
awx_1_1     | 2021-05-13 13:17:12,504 INFO success: stdout entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
awx_1_1     | 2021-05-13 13:17:12,505 INFO success: awx-dispatcher entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
awx_1_1     | 2021-05-13 13:17:12,505 INFO success: awx-receiver entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
awx_1_1     | 2021-05-13 13:17:12,505 INFO success: awx-uwsgi entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
awx_1_1     | 2021-05-13 13:17:12,505 INFO success: awx-daphne entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
awx_1_1     | 2021-05-13 13:17:12,507 INFO spawned: 'awx-nginx' with pid 165
awx_1_1     | 2021-05-13 13:17:12,507 INFO success: awx-wsbroadcast entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
awx_1_1     | 2021-05-13 13:17:12,507 INFO success: awx-rsyslogd entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
awx_1_1     | 2021-05-13 13:17:12,507 INFO success: awx-receptor entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
awx_1_1     | awx-dispatcher stdout | make[1]: Entering directory '/awx_devel'
awx_1_1     | awx-dispatcher stdout |
awx_1_1     | awx-receiver stdout | make[1]: Entering directory '/awx_devel'
awx_1_1     | awx-receiver stdout |
awx_1_1     | awx-daphne stdout | make[1]: Entering directory '/awx_devel'
awx_1_1     | awx-daphne stdout |
awx_1_1     | awx-nginx stdout | make[1]: Entering directory '/awx_devel'
awx_1_1     | awx-nginx stdout |
awx_1_1     | awx-wsbroadcast stdout | make[1]: Entering directory '/awx_devel'
awx_1_1     | awx-wsbroadcast stdout |
awx_1_1     | awx-uwsgi stdout | make[1]: Entering directory '/awx_devel'
awx_1_1     | awx-uwsgi stdout |
awx_1_1     | awx-nginx stdout | nginx -g "daemon off;"
awx_1_1     | awx-nginx stdout |
awx_1_1     | awx-nginx stdout | nginx: [emerg] getpwnam("nginx") failed
awx_1_1     | awx-nginx stdout |
awx_1_1     | awx-nginx stdout | make[1]: *** [Makefile:259: nginx] Error 1
awx_1_1     | awx-nginx stdout | make[1]: Leaving directory '/awx_devel'
awx_1_1     | awx-nginx stdout |
awx_1_1     | awx-receptor stdout | INFO 2021/05/13 13:17:11 Running control service control
awx_1_1     | awx-receptor stdout |
awx_1_1     | awx-receptor stdout | INFO 2021/05/13 13:17:11 Initialization complete
awx_1_1     | awx-receptor stdout |
awx_1_1     | awx-nginx stdout | make[1]: Entering directory '/awx_devel'
awx_1_1     | awx-nginx stdout |
awx_1_1     | awx-nginx stdout | nginx -g "daemon off;"
awx_1_1     | awx-nginx stdout |
awx_1_1     | awx-nginx stdout | nginx: [emerg] getpwnam("nginx") failed
awx_1_1     | awx-nginx stdout |
awx_1_1     | 2021-05-13 13:17:12,642 INFO exited: awx-nginx (exit status 2; not expected)
awx_1_1     | awx-nginx stdout | make[1]: *** [Makefile:259: nginx] Error 1
awx_1_1     | awx-nginx stdout | make[1]: Leaving directory '/awx_devel'
awx_1_1     | awx-nginx stdout |
awx_1_1     | awx-daphne stdout | 2021-05-13 13:17:13,382 INFO     [-] daphne.cli Starting server at tcp:port=8051:interface=127.0.0.1
awx_1_1     | awx-daphne stdout | 2021-05-13 13:17:13,382 INFO     Starting server at tcp:port=8051:interface=127.0.0.1
awx_1_1     | awx-daphne stdout | 2021-05-13 13:17:13,383 INFO     [-] daphne.server HTTP/2 support not enabled (install the http2 and tls Twisted extras)
awx_1_1     | awx-daphne stdout | 2021-05-13 13:17:13,383 INFO     HTTP/2 support not enabled (install the http2 and tls Twisted extras)
awx_1_1     | awx-daphne stdout | 2021-05-13 13:17:13,383 INFO     [-] daphne.server Configuring endpoint tcp:port=8051:interface=127.0.0.1
awx_1_1     | awx-daphne stdout | 2021-05-13 13:17:13,383 INFO     Configuring endpoint tcp:port=8051:interface=127.0.0.1
awx_1_1     | awx-daphne stdout | 2021-05-13 13:17:13,384 INFO     [-] daphne.server Listening on TCP address 127.0.0.1:8051
awx_1_1     | awx-daphne stdout | 2021-05-13 13:17:13,384 INFO     Listening on TCP address 127.0.0.1:8051
awx_1_1     | awx-daphne stdout |
awx_1_1     | awx-uwsgi stdout | *** Starting uWSGI 2.0.18 (64bit) on [Thu May 13 13:17:13 2021] ***
awx_1_1     | awx-uwsgi stdout | compiled with version: 8.3.1 20191121 (Red Hat 8.3.1-5) on 12 May 2021 14:18:22
awx_1_1     | awx-uwsgi stdout | os: Linux-4.9.221-37.el7.x86_64 #1 SMP Tue May 5 14:34:34 UTC 2020
awx_1_1     | awx-uwsgi stdout | nodename: awx_1
awx_1_1     | awx-uwsgi stdout | machine: x86_64
awx_1_1     | awx-uwsgi stdout | clock source: unix
awx_1_1     | awx-uwsgi stdout | detected number of CPU cores: 4
awx_1_1     | awx-uwsgi stdout | current working directory: /awx_devel
awx_1_1     | awx-uwsgi stdout | detected binary path: /var/lib/awx/venv/awx/bin/uwsgi
awx_1_1     | awx-uwsgi stdout | !!! no internal routing support, rebuild with pcre support !!!
awx_1_1     | awx-uwsgi stdout | uWSGI running as root, you can use --uid/--gid/--chroot options
awx_1_1     | awx-uwsgi stdout | *** WARNING: you are running uWSGI as root !!! (use the --uid flag) ***
awx_1_1     | awx-uwsgi stdout | chdir() to /awx_devel/
awx_1_1     | awx-uwsgi stdout | your processes number limit is 1048576
awx_1_1     | awx-uwsgi stdout | your memory page size is 4096 bytes
awx_1_1     | awx-uwsgi stdout |  *** WARNING: you have enabled harakiri without post buffering. Slow upload could be rejected on post-unbuffered webservers ***
awx_1_1     | awx-uwsgi stdout | detected max file descriptor number: 1048576
awx_1_1     | awx-uwsgi stdout | lock engine: pthread robust mutexes
awx_1_1     | awx-uwsgi stdout | thunder lock: disabled (you can enable it with --thunder-lock)
awx_1_1     | awx-uwsgi stdout | uwsgi socket 0 bound to TCP address 127.0.0.1:8050 fd 3
awx_1_1     | awx-uwsgi stdout | uWSGI running as root, you can use --uid/--gid/--chroot options
awx_1_1     | awx-uwsgi stdout | *** WARNING: you are running uWSGI as root !!! (use the --uid flag) ***
awx_1_1     | awx-uwsgi stdout | Python version: 3.8.3 (default, Aug 31 2020, 16:03:14)  [GCC 8.3.1 20191121 (Red Hat 8.3.1-5)]
awx_1_1     | awx-uwsgi stdout |
awx_1_1     | awx-uwsgi stdout | PEP 405 virtualenv detected: /var/lib/awx/venv/awx
awx_1_1     | awx-uwsgi stdout | Set PythonHome to /var/lib/awx/venv/awx
awx_1_1     | awx-uwsgi stdout |
awx_1_1     | awx-uwsgi stdout | Python main interpreter initialized at 0xcbade0
awx_1_1     | awx-uwsgi stdout | uWSGI running as root, you can use --uid/--gid/--chroot options
awx_1_1     | awx-uwsgi stdout | *** WARNING: you are running uWSGI as root !!! (use the --uid flag) ***
awx_1_1     | awx-uwsgi stdout | python threads support enabled
awx_1_1     | awx-uwsgi stdout | your server socket listen backlog is limited to 100 connections
awx_1_1     | awx-uwsgi stdout | your mercy for graceful operations on workers is 60 seconds
awx_1_1     | awx-uwsgi stdout | mapped 609552 bytes (595 KB) for 5 cores
awx_1_1     | awx-uwsgi stdout | *** Operational MODE: preforking ***
awx_1_1     | awx-uwsgi stdout | uWSGI running as root, you can use --uid/--gid/--chroot options
awx_1_1     | awx-uwsgi stdout | *** WARNING: you are running uWSGI as root !!! (use the --uid flag) ***
awx_1_1     | awx-uwsgi stdout | *** uWSGI is running in multiple interpreter mode ***
awx_1_1     | awx-uwsgi stdout | spawned uWSGI master process (pid: 197)
awx_1_1     | awx-uwsgi stdout | spawned uWSGI worker 1 (pid: 198, cores: 1)
awx_1_1     | awx-uwsgi stdout | spawned uWSGI worker 2 (pid: 199, cores: 1)
awx_1_1     | awx-uwsgi stdout | spawned uWSGI worker 3 (pid: 200, cores: 1)
awx_1_1     | awx-uwsgi stdout | spawned uWSGI worker 4 (pid: 201, cores: 1)
awx_1_1     | awx-uwsgi stdout | spawned uWSGI worker 5 (pid: 202, cores: 1)
awx_1_1     | awx-uwsgi stdout | *** Stats server enabled on /tmp/stats.socket fd: 17 ***
awx_1_1     | awx-uwsgi stdout |
awx_1_1     | awx-uwsgi stdout | Python auto-reloader enabled
awx_1_1     | awx-uwsgi stdout |
awx_1_1     | 2021-05-13 13:17:14,723 INFO spawned: 'awx-nginx' with pid 218
awx_1_1     | awx-dispatcher stdout | 2021-05-13 13:17:14,720 ERROR    [-] awx.main.tasks Host awx last checked in at 2021-05-13 12:15:20.862677+00:00, marked as lost.
awx_1_1     | awx-dispatcher stdout |
awx_1_1     | awx-nginx stdout | make[1]: Entering directory '/awx_devel'
awx_1_1     | awx-nginx stdout |
awx_1_1     | awx-nginx stdout | nginx -g "daemon off;"
awx_1_1     | awx-nginx stdout |
awx_1_1     | awx-nginx stdout | nginx: [emerg] getpwnam("nginx") failed
awx_1_1     | awx-nginx stdout |
awx_1_1     | 2021-05-13 13:17:14,856 INFO exited: awx-nginx (exit status 2; not expected)
awx_1_1     | awx-nginx stdout | make[1]: *** [Makefile:259: nginx] Error 1
awx_1_1     | awx-nginx stdout | make[1]: Leaving directory '/awx_devel'
awx_1_1     | awx-nginx stdout |
awx_1_1     | awx-wsbroadcast stdout | 2021-05-13 13:17:14,871 ERROR    [-] awx.main.wsbroadcast AWX is currently installing/upgrading.  Trying again in 5s...
awx_1_1     | awx-wsbroadcast stdout |
awx_1_1     | awx-receiver stdout | 2021-05-13 13:17:14,888 WARNING  [-] awx.main.commands.run_callback_receiver scaling up worker pid:224
awx_1_1     | awx-receiver stdout |
awx_1_1     | awx-receiver stdout | 2021-05-13 13:17:14,898 WARNING  [-] awx.main.commands.run_callback_receiver scaling up worker pid:226
awx_1_1     | awx-receiver stdout |
awx_1_1     | awx-receiver stdout | 2021-05-13 13:17:14,912 WARNING  [-] awx.main.commands.run_callback_receiver scaling up worker pid:227
awx_1_1     | awx-receiver stdout |
awx_1_1     | awx-receiver stdout | 2021-05-13 13:17:14,922 WARNING  [-] awx.main.commands.run_callback_receiver scaling up worker pid:228
awx_1_1     | awx-receiver stdout |
awx_1_1     | 2021-05-13 13:17:15,419 INFO waiting for awx-rsyslogd to stop
awx_1_1     | 2021-05-13 13:17:15,424 INFO stopped: awx-rsyslogd (exit status 0)
awx_1_1     | 2021-05-13 13:17:15,428 INFO spawned: 'awx-rsyslogd' with pid 229
awx_1_1     | 2021-05-13 13:17:16,431 INFO success: awx-rsyslogd entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
awx_1_1     | awx-uwsgi stdout | WSGI app 0 (mountpoint='') ready in 3 seconds on interpreter 0xcbade0 pid: 202 (default app)
awx_1_1     | awx-uwsgi stdout |
awx_1_1     | awx-uwsgi stdout | WSGI app 0 (mountpoint='') ready in 3 seconds on interpreter 0xcbade0 pid: 199 (default app)
awx_1_1     | awx-uwsgi stdout |
awx_1_1     | awx-uwsgi stdout | WSGI app 0 (mountpoint='') ready in 3 seconds on interpreter 0xcbade0 pid: 198 (default app)
awx_1_1     | awx-uwsgi stdout | running "exec: supervisorctl restart tower-processes:awx-dispatcher tower-processes:awx-receiver" (accepting1)...
awx_1_1     | awx-uwsgi stdout |
awx_1_1     | awx-uwsgi stdout | WSGI app 0 (mountpoint='') ready in 3 seconds on interpreter 0xcbade0 pid: 201 (default app)
awx_1_1     | awx-uwsgi stdout |
awx_1_1     | awx-uwsgi stdout | WSGI app 0 (mountpoint='') ready in 3 seconds on interpreter 0xcbade0 pid: 200 (default app)
awx_1_1     | awx-uwsgi stdout |
awx_1_1     | 2021-05-13 13:17:16,872 INFO waiting for awx-dispatcher to stop
awx_1_1     | 2021-05-13 13:17:16,877 INFO stopped: awx-dispatcher (terminated by SIGKILL)
awx_1_1     | 2021-05-13 13:17:16,878 INFO waiting for awx-receiver to stop
awx_1_1     | 2021-05-13 13:17:16,892 INFO stopped: awx-receiver (terminated by SIGKILL)
awx_1_1     | 2021-05-13 13:17:16,896 INFO spawned: 'awx-dispatcher' with pid 257
awx_1_1     | awx-dispatcher stdout | make[1]: Entering directory '/awx_devel'
awx_1_1     | awx-dispatcher stdout |
awx_1_1     | 2021-05-13 13:17:17,908 INFO success: awx-dispatcher entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
awx_1_1     | 2021-05-13 13:17:17,910 INFO spawned: 'awx-nginx' with pid 265
awx_1_1     | 2021-05-13 13:17:17,914 INFO spawned: 'awx-receiver' with pid 266
awx_1_1     | awx-nginx stdout | make[1]: Entering directory '/awx_devel'
awx_1_1     | awx-nginx stdout |
awx_1_1     | awx-receiver stdout | make[1]: Entering directory '/awx_devel'
awx_1_1     | awx-receiver stdout |
awx_1_1     | awx-nginx stdout | nginx -g "daemon off;"
awx_1_1     | awx-nginx stdout |
awx_1_1     | awx-nginx stdout | nginx: [emerg] getpwnam("nginx") failed
awx_1_1     | awx-nginx stdout |
awx_1_1     | 2021-05-13 13:17:17,958 INFO exited: awx-nginx (exit status 2; not expected)
awx_1_1     | 2021-05-13 13:17:17,959 INFO gave up: awx-nginx entered FATAL state, too many start retries too quickly
awx_1_1     | awx-nginx stdout | make[1]: *** [Makefile:259: nginx] Error 1
awx_1_1     | awx-nginx stdout | make[1]: Leaving directory '/awx_devel'
awx_1_1     | awx-nginx stdout |
awx_1_1     | 2021-05-13 13:17:18,960 INFO success: awx-receiver entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
awx_1_1     | awx-uwsgi stdout | tower-processes:awx-dispatcher: stopped
awx_1_1     | awx-uwsgi stdout | tower-processes:awx-receiver: stopped
awx_1_1     | awx-uwsgi stdout | tower-processes:awx-dispatcher: started
awx_1_1     | awx-uwsgi stdout | tower-processes:awx-receiver: started
awx_1_1     | awx-uwsgi stdout |
awx_1_1     | 2021-05-13 13:17:19,683 INFO waiting for awx-rsyslogd to stop
awx_1_1     | 2021-05-13 13:17:19,684 INFO stopped: awx-rsyslogd (exit status 0)
awx_1_1     | 2021-05-13 13:17:19,689 INFO spawned: 'awx-rsyslogd' with pid 295
awx_1_1     | 2021-05-13 13:17:20,224 INFO exited: awx-wsbroadcast (exit status 0; expected)
awx_1_1     | 2021-05-13 13:17:20,226 INFO spawned: 'awx-wsbroadcast' with pid 297
awx_1_1     | awx-wsbroadcast stdout | make[1]: Leaving directory '/awx_devel'
awx_1_1     | awx-wsbroadcast stdout |
awx_1_1     | awx-wsbroadcast stdout | make[1]: Entering directory '/awx_devel'
awx_1_1     | awx-wsbroadcast stdout |
awx_1_1     | awx-receiver stdout | 2021-05-13 13:17:20,369 WARNING  [-] awx.main.commands.run_callback_receiver scaling up worker pid:304
awx_1_1     | awx-receiver stdout |
awx_1_1     | awx-receiver stdout | 2021-05-13 13:17:20,375 WARNING  [-] awx.main.commands.run_callback_receiver scaling up worker pid:305
awx_1_1     | awx-receiver stdout |
awx_1_1     | awx-receiver stdout | 2021-05-13 13:17:20,380 WARNING  [-] awx.main.commands.run_callback_receiver scaling up worker pid:306
awx_1_1     | awx-receiver stdout |
awx_1_1     | awx-receiver stdout | 2021-05-13 13:17:20,385 WARNING  [-] awx.main.commands.run_callback_receiver scaling up worker pid:307
awx_1_1     | awx-receiver stdout |
awx_1_1     | 2021-05-13 13:17:21,389 INFO success: awx-wsbroadcast entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
awx_1_1     | 2021-05-13 13:17:21,389 INFO success: awx-rsyslogd entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
awx_1_1     | awx-wsbroadcast stdout | 2021-05-13 13:17:22,610 ERROR    [-] awx.main.wsbroadcast AWX is currently installing/upgrading.  Trying again in 5s...
awx_1_1     | awx-wsbroadcast stdout |
awx_1_1     | 2021-05-13 13:17:27,912 INFO exited: awx-wsbroadcast (exit status 0; expected)
awx_1_1     | 2021-05-13 13:17:27,914 INFO spawned: 'awx-wsbroadcast' with pid 314
awx_1_1     | awx-wsbroadcast stdout | make[1]: Leaving directory '/awx_devel'
awx_1_1     | awx-wsbroadcast stdout |
awx_1_1     | awx-wsbroadcast stdout | make[1]: Entering directory '/awx_devel'
awx_1_1     | awx-wsbroadcast stdout |
awx_1_1     | 2021-05-13 13:17:28,926 INFO success: awx-wsbroadcast entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
awx_1_1     | awx-wsbroadcast stdout | 2021-05-13 13:17:30,349 ERROR    [-] awx.main.wsbroadcast AWX is currently installing/upgrading.  Trying again in 5s...
awx_1_1     | awx-wsbroadcast stdout |
awx_1_1     | 2021-05-13 13:17:35,650 INFO exited: awx-wsbroadcast (exit status 0; expected)
awx_1_1     | awx-wsbroadcast stdout | make[1]: Leaving directory '/awx_devel'
awx_1_1     | awx-wsbroadcast stdout |
awx_1_1     | 2021-05-13 13:17:35,651 INFO spawned: 'awx-wsbroadcast' with pid 327
awx_1_1     | awx-wsbroadcast stdout | make[1]: Entering directory '/awx_devel'
awx_1_1     | awx-wsbroadcast stdout |
awx_1_1     | 2021-05-13 13:17:36,663 INFO success: awx-wsbroadcast entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
awx_1_1     | awx-wsbroadcast stdout | 2021-05-13 13:17:38,080 ERROR    [-] awx.main.wsbroadcast AWX is currently installing/upgrading.  Trying again in 5s...
```

**Comment by jbradberry** [comment]
The fix for this landed in 19.3.0, from PR #10633.  It looks to me like the underlying problem was introduced in 19.1.0.

**Comment by mabashian** [comment]
@jbradberry since that fix merged I'm gonna close this
